docs/docs/mapping/patch.md

```diff
@@ -94,7 +94,7 @@ that is what we specify in the field_mask.
 
 ```shell
 $ curl \
-	--data '{"abe":{"single_nested":{"amount":457},"string_value":"some value that won't get updated because not in the field mask"},"update_mask":{"paths":["single_nested"]}}' \
+	--data '{"abe":{"single_nested":{"amount":457},"string_value":"some value that will not get updated because not in the field mask"},"update_mask":"singleNested"}}' \
```

Comment:
Shouldn't this be `single_nested`? I generally use the proto field name casing to identify fields in the mask.
```suggestion
	--data '{"abe":{"single_nested":{"amount":457},"string_value":"some value that will not get updated because not in the field mask"},"update_mask":"single_nested"}}' \
```

Reply:
This is unfortunate but ... expects paths to be in the `smallCamelCase`.

> …
> Fields name in each path are converted to/from lower-camel naming conventions.
> …

https://github.com/protocolbuffers/protobuf-go/blob/0fd4f3a506894c05a9cad2d52c521a8ce953b49d/encoding/protojson/well_known_types.go#L837-L840


Using `snake_case` leads to error:
```txt
{"code":3, "message":"proto: (line 1:136): google.protobuf.FieldMask.paths contains invalid path: \"single_nested\"", "details":[]}
```

Reply:
TIL! That makes sense on some level - the official protobuf json convention is to use lower camelCase. We should probably update the rest of the fields to use lower camel case instead. What do you think?

Reply:
Agree. Changed.